Assignment/Assignment/SampleData.cs

```diff
@@ -1,30 +1,95 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 
 namespace Assignment
 {
     public class SampleData : ISampleData
     {
         // 1.
-        public IEnumerable<string> CsvRows => throw new NotImplementedException();
+        public IEnumerable<string> CsvRows { get => File.ReadAllLines("People.csv").Skip(1).ToList(); }
 
         // 2.
-        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows() 
-            => throw new NotImplementedException();
+        public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
+        {
+            //Create the query
+            IEnumerable<string> stateQuery =
+                from line in CsvRows
+                let state = line.Split(',')
+                orderby state[6]
+                select state[6];
+
+            //Execute the query
+            var stateList = stateQuery.Distinct().ToList();    
+
+            return stateList;

```

Comment:
```suggestion
          return CsvRows
                    .Select(item => item.Split(',')[6])
                    .Distinct()
                    .OrderBy(item => item);
```
It's a bit unexpected to use the query flavor of the LINQ syntax - but if you do I would not recommend mixing the two flavors.

Above is the way to go about it with the method syntax, which is a bit more C# looking and, in my opinion, fits better in your code in terms of readability.

Additionally, you do not need to convert this to a list, as the method returns an `IEnumerable`. Although I like your `Execute Query` comment, but it may be best to continue delaying the execution and return the query itself rather than it's executed result.